README.md

```diff
@@ -562,7 +562,7 @@ terraform:
 
 ### CLI interface
 
-#### maas-anvil [OPTIONS] COMMAND [ARGS]...
+#### maas-anvil [OPTIONS] COMMAND [ARGS]
```

Comment:
nit: let's revert this even though the linter is complaining. This is to indicate that the command is expecting one or more [ARGS]